### PR TITLE
Parse blob url

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

**Comment by dagood** [comment]
> Add DeleteBlobs target which deletes all blobs under a specific blob path

Will this be on the VSTS side? Not seeing a new target in this PR.

**Comment by mmitche** [comment]
@dagood Actually it's in the orchestration utilities repo.  This just needs to go in first.

**Comment by mmitche** [comment]
@dagood Yeah I was working on both repos at once and mixed the commit messages

File: src/Microsoft.DotNet.Build.Tasks.Feed/PackageFiles/Microsoft.DotNet.Build.Tasks.Feed.targets [hunk 1]
```diff
@@ -125,15 +125,15 @@
       <ManifestFile Condition="'$(ManifestFile)' == ''">$(ManifestTempDir)build.xml</ManifestFile>
     </PropertyGroup>
 
-    <ParseFeedUrl FeedUrl="$(ExpectedFeedUrl)">
+    <ParseBlobUrl BlobUrl="$(ExpectedFeedUrl.Replace('/index.json', '')">
```
Comment (dagood) [inline src/Microsoft.DotNet.Build.Tasks.Feed/PackageFiles/Microsoft.DotNet.Build.Tasks.Feed.targets]: I was working on merging this into my changes and noticed the parens are unbalanced here.

**Comment by mmitche** [comment]
@dagood Apparently msbuild didn't care...

**Pull request merged**